features/projects/components/project-list/error-page/error-page.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import {
+  AlertIcon,
+  AlertWrapper,
+  ArrowIcon,
+  Container,
+  Message,
+  RefreshButton,
+  Wrapper,
+} from "./error-style";
+
+export function ErrorPage() {
+  const refreshPage = () => {
+    window.location.reload();
```

Comment:
As an alternative you can also use the Next.js router to refresh the page: https://nextjs.org/docs/api-reference/next/router#routerreload

Reply:
Tbh I'm not sure what the advantage though

Reply:
Wow! haven't used next js router, this is new to me, but excited to do it,, Thanks!! you're the man!😎